### PR TITLE
docs: multiple changes

### PR DESCRIPTION
**Pull request by ReignOfComputer**
- Team logo now displays correctly on PPP via absolute pathing
- Images and captions are now centralized
- Removed TODO and stubs from UG/DG
- Removed Tutor Portfolio link on AboutUs
- My PPP now magically prints to 10 pages through @media print, recommend you guys do the same

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -904,8 +937,6 @@ Use case ends.
 +
 Use case ends.
 
-{*TODO* : Please add use cases if your component is not featured here yet. If unsure, just put to be safe. *DELETE WHEN DONE*}
```
Comment (Q-gabe) [inline docs/DeveloperGuide.adoc]: Thanks for the cleanup!

Comment (ReignOfComputer) [reply docs/DeveloperGuide.adoc]: :)

File: docs/team/reignofcomputer.adoc [hunk 2]
```diff
@@ -166,18 +175,26 @@ These operations are exposed in the `Model` interface.
...
 image::DGApptAddAppointmentSequenceDiagram.png[]
 
 The following activity diagram shows how adding an appointment works, using follow-ups as an example:
 
+[.text-center]
 .Adding an Appointment Activity Diagram
-image::DGApptFollowUpActivityDiagram.png[]
+image::DGApptFollowUpActivityDiagram.png[width="275"]
 
+[.hide-for-print]
```
Comment (Q-gabe) [inline docs/team/reignofcomputer.adoc]: 😉

Comment (ReignOfComputer) [reply docs/team/reignofcomputer.adoc]: *hides*

**Pull request merged**